guides/common/modules/proc_using-ansible-provider-for-package-and-errata-actions.adoc

```diff
@@ -0,0 +1,17 @@
+[id="Using_Ansible_Provider_for_Package_and_Errata_Actions_{context}"]
+= Using Ansible provider for package and errata actions
+
+By default, {Project} is configured to use the Script provider templates for remote execution jobs.
+If you prefer using Ansible job templates for your remote jobs, you can configure {Project} to use them by default for remote execution features associated with them.
+
+.Prerequisites
+* Use remote execution in `ssh` mode.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Administer* > *Remote Execution Features*.
+. Find each feature whose name contains `by_search`.
+. Change the job template for these features from `Katello Script Default` to `Katello Ansible Default`.
+. Click *Submit*.
+
+Ansible job templates are now used for remote execution when using the {ProjectWebUI} to perform package and errata actions.
+They are also used when using `hammer job-invocation create` with the same remote execution features used in the procedure above.
```

Comment:
```suggestion
{Project} now uses Ansible provider templates for remote execution jobs by which you can perform package and errata actions.
This applies to job invocations from the {ProjectWebUI} as well as by using Hammer CLI, with the same remote execution features that you have changed.
```
Perhaps something like this?

Reply:
Why avoid naming `hammer job-invocation create`? I guess it's fine but I feel like the original was easier to read. (I am biased, though.. ;)

Reply:
I think that level of detail is not necessary in this place, but if @bangelic thinks it's better to include it, I'm not against it.

Reply:
I'll go ahead and keep `hammer job-invocation create` in there.